migrations/2021-02-14-201950_session-table/up.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE sessions
+(
+    id SERIAL
+        CONSTRAINT sessions_pk
+            PRIMARY KEY,
+    user_id INTEGER NOT NULL
+        CONSTRAINT sessions_users_id_fk
+            REFERENCES users
+            ON UPDATE CASCADE ON DELETE CASCADE,
```

Comment:
You'll probably want to delete stale sessions eventually. In a big database, deleting old rows can be relatively expensive. For this, you might want to use partitioning (https://www.postgresql.org/docs/10/ddl-partitioning.html), which lets you drop old, stale partitions cheaply while still treating the partitions as a single table. However, to use partitioning you need to have no foreign keys on the partitioned table. Suggestion: Make user_id not a foreign key. We don't expect that user_ids change, so we don't need `ON UPDATE CASCADE`. And if the users entry is deleted, we can simply treat the session as invalid. So we don't need `ON DELETE CASCADE`.